js/persistence/MessagePersistence.coffee

```diff
@@ -17,10 +19,14 @@ module.exports = ({MessageActions}) ->
         if res.data.ok is true
           MessageActions.listeningStation station
         if res.data?.circle?.nes # prize
-          res.data.circle.nes.map (env) ->
-            env.gam.heard = true
-            env
-          MessageActions.loadMessages res.data.circle.nes
+          if (res.data.circle.nes.length == 0) and (typeof since == "object")
```

Comment:
Dispatching based on the type of the date feels pretty weird to me. I think an explicit boolean flag would be cleaner.

Reply:
Actually, this check is redundant. This function is only ever called with Date objects, so I should just do away with the type-check.